### PR TITLE
Add departures in status with some attributes added in all objects

### PR DESCRIPTION
**Pull request by kadhikari**
The followings are the modifications:
1. departures add in status
2. Some attributes added in each object of status
- source_status: reflects status of the external service used by the connector to feed api (values = ok/ko)
- last_status_update: latest date-time when connector was called to verify it's status
- message: Message if exists when the external service was called.  

For more details: https://navitia.atlassian.net/browse/NAV-1693

### CODE REVIEW

File: internal/departures/context.go [hunk 1]
```diff
@@ -11,10 +11,13 @@ import (
 type DeparturesContext struct {
 	departures          *map[string][]Departure
 	lastDepartureUpdate time.Time
+	lastStatusUpdate    time.Time
```
Comment (pbench) [inline internal/departures/context.go:14]: Unless i miss something, I think  this variable is never set/updated

Comment (kadhikari) [reply internal/departures/context.go:14]: As this one others also will be updated in future for each api one by one.

File: api/api.go [hunk 2]
```diff
@@ -106,19 +117,28 @@ func StatusHandler(manager *manager.DataManager) gin.HandlerFunc {
...
 		}
 
 		var lastEquipmentDataUpdate time.Time
 		if manager.GetEquipmentsContext() != nil {
 			lastEquipmentDataUpdate = manager.GetEquipmentsContext().GetLastEquipmentsDataUpdate()
 		}
 
-		var lastDeparturesDataUpdate time.Time
+		var lastDeparturesDataUpdate, lastDeparturesStatusUpdate time.Time
+		var loadDeparturesData bool = true
```
Comment (SGrenet) [inline api/api.go:133]: not in manager?

Comment (kadhikari) [reply api/api.go:133]: We don't have any mechanic to active/inactive, so this value will be displayed as `true`

**Review by pbench — APPROVED** [review]

**Review by pbench — APPROVED** [review]

**Pull request merged**